kaisen/kaisen/src/taiwo/nested_function.py

```diff
@@ -0,0 +1,11 @@
+def call(a,b):
```

Comment:
Awesome, but call is a keyword in python you can changed it to `add_sum()`

Reply:
Okay